plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -340,14 +340,14 @@ object ScalafmtPlugin extends AutoPlugin {
         sbtConfig.value,
         streams.value.log,
         outputStreamWriter(streams.value),
-        fullResolvers.value
+        resolvers.value
```

Comment:
this partially reverts https://github.com/scalameta/sbt-scalafmt/pull/87

since there is no test, it's hard to know what part of https://github.com/sbt/sbt/blob/83fc304d608d63b9742d3d760ac472433ed82be5/main/src/main/scala/sbt/Defaults.scala#L2255-L2276 is useful, but I wonder if it's really required since it seems to be very project-specific as opposed to build-specific?

Reply:
It's dangerous change. Is it really necessary? #87 fixed plugin fetching for many people with corporate proxy.

Reply:
Apart from `resolvers`, what the implementation of `fullResolvers` depends on is scoped to the project-level, so as we scope `scalafmtSbt` to `ThisBuild`, we can't access them any more.

@slivkamiro are your custom resolvers defined at the project level? I browsed though the discussion but I could not find what explicit scenario that #87 helped with.

Reply:
Hi, we have our repositories defined in a configuration file and we are passing `-Dsbt.repository.config` to build command.